tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -1171,9 +1145,8 @@ def test_work_order_with_response_timeout_str (self):
             self.test_obj.build_request_output['request_obj'])
 
         assert (
-                verify_test(
-                    result_response, 0,
-                    self.test_obj.build_request_output['pre_test_output'],
-                    self.test_obj.build_request_output['action_obj'])
+                check_negative_test_responses(
```

Comment:
Similar to above.

Reply:
"Invalid data format for responseTimeoutMSecs" is the new error introduced by Ram. hence change is valid